src/utilities/set-root-property.ts

```diff
@@ -1,13 +1,10 @@
 export function setRootProperty(
   name: string,
   value: string,
-  node: Element | null = null,
+  node?: HTMLElement,
 ) {
-  if (document == null) {
-    return;
-  }
+  if (!document) return;
 
-  const styleNode =
-    node && node instanceof HTMLElement ? node : document.documentElement;
-  styleNode && styleNode.style.setProperty(name, value);
+  const styleNode = node ? node : document.documentElement;
+  styleNode.style.setProperty(name, value);
```

Comment:
**suggestion(non-blocking):** Thoughts on naming this [`element`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style#syntax)? `styledNode` makes me think it's an instance of `document.querySelector('style')`
```suggestion
  const element = node || document.documentElement;

  element.style.setProperty(name, value);
```